report/src/utils.ts

```diff
@@ -44,7 +44,7 @@ export const pct = (part: number, total: number) => {
   return (part / total) * 100;
 };
 
-export const formatPct = (percentage: number) => {
+export const formatPct = (percentage: number, hideSymbol = false) => {
```

Comment:
I don't quite understand why do you want to hide the % symbol. Could you please help me here?

I find the part of the ui without "%" captured in the screenshot puzzling. I suggest you remove this part and use % everywhere.

Reply:
I initially had it and then it felt a bit unnecessary and took up space.

I wavered a bit on whether to drop it or not, so it's good feedback if it feels off to you. I can revert this bit.